src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -400,14 +460,37 @@ const WorkflowView = _.flow(
     }
   }
 
+  describeSelectionModel() {
+    const { modifiedConfig: { rootEntityType }, entityMetadata, entitySelectionModel: { newSetName, selectedEntities, type } } = this.state
+    const { name } = selectedEntities // entityType?
+    const count = _.size(selectedEntities)
+    const newSetMessage = count > 1 ? `(will create a new set named "${newSetName}")` : ''
+    return Utils.cond(
+      [this.isSingle() || !rootEntityType, ''],
+      [type === EntitySelectionType.processAll, `all ${entityMetadata[rootEntityType].count} ${rootEntityType}s (will create a new set named "${newSetName}")`],
```

Comment:
This errors out the whole app when `entityMetadata[rootEntityType]` is undefined.

Reply:
You're right. Thanks for finding this. I was so busy testing selection of set types that I forgot to go back and test the other radio button state. I will fix.